src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,35 @@
+.card__rating {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 16px;
+  width: 100%;
+  gap: 4px;
+
+  .stars {
+    display: flex;
+    gap: 4px;
+
+    &__star {
+      background-image: url(../images/star.svg);
+      width: 16px;
+      height: 16px;
+      background-position: center;
+      background-repeat: no-repeat;
+    }
+
+    &--4 {
+      .stars__star:nth-child(-n + 4) {
+        background-image: url(../images/star-active.svg);
+      }
+    }
+  }
+
+  .card__reviews {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $color-tertiary;
```

Comment:
Ensure that the variable `$color-tertiary` is defined in your SCSS files to avoid compilation errors.